File: android/src/main/java/br/com/gs3tecnologia/launchnavigator/LaunchNavigatorPlugin.java
package br.com.gs3tecnologia.launchnavigator;

import com.getcapacitor.Plugin;
import com.getcapacitor.PluginCall;
import com.getcapacitor.PluginMethod;
import com.getcapacitor.annotation.CapacitorPlugin;

@CapacitorPlugin(name = "LaunchNavigator")
public class LaunchNavigatorPlugin extends Plugin {
    @PluginMethod
    public void navigate(PluginCall call) {
        Double latitude = call.getDouble("latitude");
        Double longitude = call.getDouble("longitude");
        call.resolve();
    }
}

File: ios/Plugin/LaunchNavigatorPlugin.swift
import Foundation
import Capacitor

/**
 * Please read the Capacitor iOS Plugin Development Guide
 * here: https://capacitorjs.com/docs/plugins/ios
 */
@objc(LaunchNavigatorPlugin)
public class LaunchNavigatorPlugin: CAPPlugin {
    @objc func navigate(_ call: CAPPluginCall) {
        let latitude = call.getDouble("latitude") ?? 0.0
        let longitude = call.getDouble("longitude") ?? 0.0
        call.resolve()
    }
}
